### PR TITLE
Release Notes for WebView2 119 Release SDK & 120 Prerelease SDK

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article sections for review:
* https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes?branch=pr-en-us-2882&tabs=dotnetcsharp#10215140
* https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes?branch=pr-en-us-2882&tabs=dotnetcsharp#102194-prerelease

AB#47132375

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -138,6 +138,159 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
...
+
+
+<!-- ====================================================================== -->
+## 1.0.2194-prerelease
+
+Release Date: November 6, 2023
+
+[NuGet package for WebView2 SDK 1.0.2194-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2194-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 121.0.2194.0 or higher.
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/release-notes.md:231]: It would be 120.0.2194.0 this time, as that's what made the cut off date

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:231]: For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 120.0.2194.0 or higher.

**Review by victorhuangwq — APPROVED** [review]
LGTM

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -138,6 +138,159 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
...
+
+---
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+* Fixed an issue where WebView2 would sometimes render blurry content or no content after changing monitor scale or switching between RDP and docking modes.  (Runtime-only)
+* Fixed an issue in `TextServicesFoundation` causing a crash when a WebView2 instance was destroyed.  (Runtime-only)
+* Fixes a memory leak in .NET when web messages are sent from WebView2, but aren't read from the application side.  (Runtime-only)  ([Issue #3794](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3794))
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:288]: This is a fix for .NET in client DLL and .NET Core dll, so it's both SDK and runtime.

**Review by peiche-jessica — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: changed to (Runtime and SDK)

**Pull request merged**